README.md

```diff
@@ -1,40 +1,67 @@
 # ZenDesk App
 
-## Quick Start
+## Development Repository Branches Needed
 
-- clone this repo
-- cp `.env.sample` `.env`
-  - add your credentials to the `.env` file
-    - ZENDESK_URL - URL to zendesk account
-    - ZENDESK_USERNAME - your zendesk login username
-    - ZENDESK_API_TOKEN - generated via the following steps
-      - zendesk > Admin > CHANNELS > API > Settings
-      - Token Access > Enable
-      - Add API Token
+The current master branches have not been merged to work with this app and the following are needed until they are merged with master
 
-- `npm start` - start the node server
-- `npm run build` - builds the dist dir.  (TODO: can this be automated?  Fails if this is not run before `npm start`)
-- `npm run build:watch` (in separate shell) - watch for changing files and report typescript errors
+mm-webapp: `feature/cloud-apps`  
+mm-plugin-apps: `master`
 
-**Current Branches Needed**
+## Quick Start
 
-The current master branches have not been merged to work with this app and the following are needed until they are mergd with master
+### Zendesk and Mattermost Users (System Admin privileges required)
+
+1. Clone this repo
+1. cp `.env.sample` `.env`
+    1. `ZENDESK_URL` - set URL to your zendesk account host
+        1. Ex. `https://<subdomain>.zendesk.com`
+    1. `ZENDESK_CLIENT_SECRET` - (will be set later in the setup)
+1. Create Zendesk Oauth Client for Mattermost (in Zendesk)
+    1. `Zendesk` > `Admin` > `API` > `OAuth Clients`
+    1. `Add OAuth Client`
+        1. `Client Name`: `Mattermost Zendesk App`
+        1. `Description`: `Connect your Zendesk to accout to Mattermost`
```

Comment:
to rephrase?

Reply:
Thanks. rephrased as the following:
`Connect your Zendesk account to Mattermost`